src/components/tasks/tasks.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import taskData from "../../jsonData/tasks";
+import { Button, Tab, Tabs, Form } from "react-bootstrap";
+import { MdInfoOutline } from "react-icons/md";
+import { FiMoreVertical } from "react-icons/fi";
+import "./tasks.scss"
+
+class Tasks extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
Why does this component need to have state? I think, we are not handling state anywhere in it. If its not stateful consider turning it into a functional component.

Reply:
It didn't had state coz it was not connected to backend, data was hardcoded back then. ;D